tests/ctst/world/Zenko.ts

```diff
@@ -99,6 +106,326 @@ export default class Zenko extends World {
+                } else {
+                    throw new Error('Failed to generate account access keys');
+                }
+            }
+            CacheHelper.AccountName = CacheHelper.parameters.AccountName || Constants.ACCOUNT_NAME;
+        } else {
+            CacheHelper.parameters.AccessKey = CacheHelper.accountAccessKeys?.id;
+            CacheHelper.parameters.SecretKey = CacheHelper.accountAccessKeys?.value;
+        }
+    }
+
+    /**
+     * Creates an root user with policy and access keys to be used in the tests.
+     * The IAM user is cached for future tests to reduce the overall test suite
+     * duration.
+     * @returns {undefined}
+     */
+    async prepareRootUser() {
+        Zenko.IAMUserName = Zenko.IAMUserName || `${this.parameters.IAMUserName || 'usertest'}${Utils.randomString()}`;
+        Zenko.IAMUserPolicyName = `IAMUserPolicy-${Zenko.IAMUserName}${Utils.randomString()}`;
+        if (!this.cliMode.parameters.IAMSession) {
+            // Create IAM user
+            this.addCommandParameter({ userName: Zenko.IAMUserName });
+            await IAM.createUser(this.getCommandParameters());
+            this.resetCommand();
+            // Create policy
+            this.addCommandParameter({ policyName: Zenko.IAMUserPolicyName });
+            this.addCommandParameter({ policyPath: '/' });
+            if (process.env.POLICY_DOCUMENT) {
+                this.addCommandParameter({ policyDocument: JSON.parse(process.env.POLICY_DOCUMENT) });
+            } else {
+                this.addCommandParameter({ policyDocument: IAMUserPolicy });
+            }
+            const policy = await IAM.createPolicy(this.getCommandParameters());
+            const account = await SuperAdmin.getAccount({
+                name: this.parameters.AccountName || Constants.ACCOUNT_NAME,
+            });
+            let policyArn = `arn:aws:iam::${account.id}:policy/IAMUserPolicy-${Zenko.IAMUserName}}`;
+            try {
+                policyArn = JSON.parse(policy.stdout).Policy.Arn;
+            } catch (err: any) {
+                process.stderr.write(`Failed to create the IAM User policy.\n
+${JSON.stringify(policy)}\n${err.message}\n`);
+            }
+            this.resetCommand();
+            // Attach user policy
+            this.addCommandParameter({ userName: Zenko.IAMUserName });
+            this.addCommandParameter({ policyArn });
+            // Save the attached policy for cleanup
+            Zenko.IAMUserAttachedPolicy = policyArn;
+            await IAM.attachUserPolicy(this.getCommandParameters());
+            this.resetCommand();
+            // Create credentials for the user
+            this.addCommandParameter({ userName: Zenko.IAMUserName });
+            const accessKey = await IAM.createAccessKey(this.getCommandParameters());
+            this.parameters.IAMSession = JSON.parse(accessKey.stdout).AccessKey;
```

Comment:
Do we want to properly handle errors here? If the previous API call fails for some reason, the error message might be misleading here (and in IAM)